.github/workflows/be_review_prs.yml

```diff
@@ -0,0 +1,72 @@
+name: Require backend-review-group approval
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  check-approval-requirements:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request_review' || github.event.review.state == 'approved'
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get PR details
+        id: pr_details
+        uses: octokit/request-action@v2.x
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number || github.event.pull_request_review.pull_request.number }}/requested_reviewers
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check required reviewers
+        id: check_reviewers
+        run: |
+          REQUESTED_TEAMS=$(echo '${{ steps.pr_details.outputs.data }}' | jq -r '.teams[].slug')
+          if echo "$REQUESTED_TEAMS" | grep -qiE '(mobile|lighthouse|identity)'; then
```

Comment:
Can you test that these are excluded?

Reply:
Yes, I did earlier and the CI workflow passed. I'll double check again since some code was modified.